centaur/src/main/resources/standardTestCases/common_options/cache_read_off.options

```diff
@@ -0,0 +1,4 @@
+{
+  "read_from_cache": false,
+  "write_to_cache": true
```

Comment:
I like the idea of common options files, but perhaps the naming could also reflect the "write on" component as well? Not only is that explicitly turned on but it seems to be a requirement for the caching tests to work as expected. 🙂

Reply:
Makes sense, I pushed up the change. Added bonus-- it lets me try running the tests again. :)